create package for branded icons from react-icons-mdl2 package (#16655)
microsoft/fluentui: packages/react-icons-mdl2-branded/README.md

# @fluentui/react-icons-mdl2-branded

> **React Icons Mdl2 Branded components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**

Reviewer: ```suggestion
**Branded icon components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
```

> This package is to hold the branded icons from the MDL2 icon set

Reviewer: Can you add a sentence about how these icons are under the assets license?

> The `react-icons-mdl2-branded` package is released under the Microsoft Fabric Assets License. This package is to hold the branded icons from the MDL2 svg icon set

Reviewer: ```suggestion
This package contains branded icons from the MDL2 SVG icon set. Usage of these icons is subject to the [Microsoft Fabric Assets License](https://aka.ms/fluentui-assets-license).
```
